setup/gcp/installer/forseti_server_installer.py

```diff
@@ -47,12 +49,22 @@ def __init__(self, **kwargs):
         """
         super(ForsetiServerInstaller, self).__init__()
         self.config = ServerConfig(**kwargs)
+        self.v1_config = None
 
     def preflight_checks(self):
         """Pre-flight checks for server instance"""
 
         super(ForsetiServerInstaller, self).preflight_checks()
-
+        _, zone, name = gcloud.get_forseti_v1_info()
+        if name is not None:
+            # v1 instance exists, ask if the user wants to port
+            # the conf/rules settings from v1.
+            self.should_migrate_v1_configs()
```

Comment:
I think this method is more self-documenting as `self.prompt_whether_v1_configs_should_be_migrated()` or `self.prompt_v1_configs_migration()`

Reply:
Updated to prompt_v1_configs_migration()